pulsar-flink-connector/src/main/java/org/apache/flink/streaming/connectors/pulsar/table/DynamicPulsarDeserializationSchema.java

```diff
@@ -288,16 +288,16 @@ private void emitRow(@Nullable GenericRowData physicalKeyRow, @Nullable GenericR
                     rowKind,
                     physicalArity + metadataArity);
 
-            for (int keyPos = 0; keyPos < keyProjection.length; keyPos++) {
-                assert physicalKeyRow != null;
-                producedRow.setField(keyProjection[keyPos], physicalKeyRow.getField(keyPos));
-            }
-
             if (physicalValueRow != null) {
                 for (int valuePos = 0; valuePos < valueProjection.length; valuePos++) {
                     producedRow.setField(valueProjection[valuePos], physicalValueRow.getField(valuePos));
                 }
             }
+            
+             for (int keyPos = 0; keyPos < keyProjection.length; keyPos++) {
+                assert physicalKeyRow != null;
+                producedRow.setField(keyProjection[keyPos], physicalKeyRow.getField(keyPos));
+            }
```

Comment:
Please use the command `mvn license:check checkstyle:check` to check your code, you have a messed up style here.
```
2021-03-25T09:41:53.1187470Z [INFO] --- maven-checkstyle-plugin:2.17:check (checkstyle) @ pulsar-flink-connector-2.11-1.12 ---
2021-03-25T09:41:56.3645269Z [INFO] There is 1 error reported by Checkstyle 8.18 with /tools/maven/checkstyle.xml ruleset.
2021-03-25T09:41:56.3716041Z [ERROR] src/main/java/org/apache/flink/streaming/connectors/pulsar/table/DynamicPulsarDeserializationSchema.java:[304] (regexp) RegexpSingleline: Trailing whitespace
```

Reply:
done